ext/intl/calendar/gregoriancalendar_methods.cpp

```diff
@@ -87,7 +87,6 @@ static void _php_intlgregcal_constructor_body(
 				"|z!s!", &tz_object, &locale, &locale_len) == FAILURE) {
 			if (!is_constructor) {
 				zval_ptr_dtor(return_value);
-				RETVAL_NULL();
```

Comment:
This needs to stay, otherwise you may get double frees. Same below.

Reply:
Or alternatively, you can drop the whole `if (!is_constructor) { ... }` block.

Reply:
I've done this as part of https://github.com/php/php-src/commit/e0be51d0825bf91a9e9aafb9aeef165ee710bdbc.

Reply:
Thanks! It is much better what I could have done ^^ Now, I am rebasing and fixing the other issue.

Reply:
Done (although there is a test issue with `fileowner` on ARM).